pac4j-core/src/main/java/org/pac4j/core/context/J2ERequestContext.java

```diff
@@ -79,9 +80,14 @@ public Object getSessionAttribute(final String name) {
     }
 
     /**
-     * Return the request method : GET, POST...
-     *
-     * @return the request method
+     * {@inheritDoc}
+     */
+    public void invalidateSession() {
+        this.request.getSession().invalidate();
```

Comment:
On the `request`? Because the `getSession()` method will automatically create a session and return a not null object.